rasa/core/actions/forms.py

```diff
@@ -311,6 +311,17 @@ def extract_other_slots(
                 slot_mappings = self.get_mappings_for_slot(slot, domain)
 
                 for slot_mapping in slot_mappings:
+                    condition_dict = slot_mapping.get("condition")
```

Comment:
(this is [Hungarian notation btw](https://en.wikipedia.org/wiki/Hungarian_notation))